streaming/src/test/scala/org/apache/spark/streaming/DStreamClosureSuite.scala

```diff
@@ -29,24 +29,14 @@ import org.apache.spark.util.ReturnStatementInClosureException
 /**
  * Test that closures passed to DStream operations are actually cleaned.
  */
-class DStreamClosureSuite extends SparkFunSuite with BeforeAndAfterAll {
-  private var ssc: StreamingContext = null
+class DStreamClosureSuite extends SparkFunSuite with LocalStreamingContext with BeforeAndAfterAll {
```

Comment:
This test suites changed initializing SparkContext and StreamingContext "per suite" to "per test" - checking elapsed times...